deployment/scripts/uuid_migrate.sh

```diff
@@ -50,7 +50,8 @@ function create_context() {
 }
 
 function recreate_admin_certs(){
-
+  # lets not broadcast what our keys are for security reasons...
+  set +x
```

Comment:
Will this always be run with -x?  If you don't want to make that assumption, you can do this::

    usingx=$(echo $- | grep x)
    if [ -n "$usingx" ] ; then
      set +x
    fi
    ... # no trace here
    if [ -n "$usingx" ] ; then
      set -x
    fi